curvefs/src/client/warmup/warmup_manager.cpp

```diff
@@ -480,7 +482,7 @@ void WarmupManagerS3Impl::WarmUpAllObjs(
                         VLOG(9) << "no such warmup progress: " << key;
                     }
                 }
-                PutObjectToCache(context->key, context->buf, context->len);
+                PutObjectToCache(key, context->key, context->buf, context->len);
```

Comment:
> Should L482 directly return?
> 
> And, since you have found corresponding progress in L478, can we reuse the results? So that, you don't have to search again in `PutObjectToCache`?

fix